test/parallel/test-stream-pipe-await-drain.js

```diff
@@ -23,8 +23,12 @@ writer1._write = common.mustCall(function(chunk, encoding, cb) {
 }, 1);
 
 writer1.once('chunk-received', function() {
-  assert.strictEqual(reader._readableState.awaitDrain, 0,
-                     'initial value is not 0');
+  assert.strictEqual(
+    reader._readableState.awaitDrain,
+    0,
+    'awaitDrain initial value should be 0, actual is ' +
+    `${reader._readableState.awaitDrain}`
```

Comment:
Since there's only a variable in the template, you can just use the variable with no template at all:

```js
    reader._readableState.awaitDrain
```